lib/cmds/files_cmds/upload.js

```diff
@@ -67,6 +67,13 @@ exports.builder = yargs => {
       alias: 'i',
       type: 'boolean',
       default: false
+    }).option('remote-path', {
+      describe: 'Path to store this file on the PHC files for this dataset.',
+      type: 'string'
+    }).option('strip-path', {
+      describe: 'Strip local path from remote file path. This allows you to reference local a local path but removes it in the upload path.',
```

Comment:
Looks an extra `local` in this description.

Reply:
removed